SPECS/azurelinux-sysinfo/collect-sysinfo

```diff
@@ -168,7 +168,7 @@ def collect_resource_utilization():
 def collect_package_info():
     print("Collecting package info...")
     get_package_list = subprocess.run(
-        ["rpm", "-qa"], capture_output=True, text=True, check=True
```

Comment:
removing check=True means we fail on non-zero exist code - is that intentional?

Reply:
yes exactly (my comment had a typo, I mean `we won't fail`). Is that a desired state? 
IMO, there should be mandatory fields and optional fields. For mandatory, we should fail if command failed while for things like `cloud-init blame` , we can ignore exit-code.

Reply:
Okay, added back check for now and will think of how to handle failures in a separate task